src/libponyc/options/options.c

```diff
@@ -197,8 +197,7 @@ static void usage(void)
     "  --ponythreads    Use N scheduler threads. Defaults to the number of\n"
     "                   cores (not hyperthreads) available.\n"
     "  --ponyminthreads Minimum number of active scheduler threads allowed.\n"
-    "                   Defaults to 1.  Use of 0 is considered experimental\n"
-    "                   and may cause improper scheduler behavior.\n"
+    "                   Defaults to 0.\n"
```

Comment:
Can we explain what zero means here?

It's not immediately clear to me and the same will likely be true for others.

Reply:
@jemc The `0` means that it allows all scheduler threads to suspend resulting in a more efficient application with no busy polling at all when there is no work to be done (the ASIO thread is still awake and will wake up a scheduler thread when more work needs to be done either due to a timer, or network io, or some other similar event). `--ponyminthreads` controls the minimum number of scheduler threads that must be active (and busy polling for work), no matter what.

I'm not sure if you meant for me to include what `0` means in the help text. If so, I'm not sure of a succinct way to write that up.

Reply:
`Defaults to 0, meaning that all scheduler threads are allowed to be suspended when no work is available`